rln-js/index.html

```diff
@@ -127,7 +138,8 @@
 
     generateCredsButton.onclick = () => {
         membershipKey = rlnInstance.generateMembershipKey()
-        updateFields()
+        membershipId = membershipIdInput.value >= 0 ? membershipIdInput.value : 0;
```

Comment:
is that necessary? I imagine if you generate the creds then you need to get the membership id once you have inserted them in the contract, right?